archipelago/sta_delays.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Can you make sure if this file is packaged properly when you do `python setup.py bdist_wheel`?

Reply:
I added sta_delays to setup.py that is now packaged when running `python setup.py bdist_wheel`

Reply:
I've never used bdist_wheel before though so let me know if I did it wrong

Reply:
You forgot the `include_package_data = True`. I've pushed the changes.